tests/Spec.hs

```diff
@@ -372,6 +372,39 @@ main = hspec $ do
         , ALeaveComment (PullRequestId 3) "Pull request approved by @deckard, waiting for rebase behind 2 pull requests."
         ]
 
+    it "abandons integration candidates closed by user" $ do
```

Comment:
I would say “when a pull request is closed” instead of “closed by the user” to clarify what event triggers this.